src/quacc/recipes/common/phonons.py

```diff
@@ -71,59 +73,51 @@ def phonon_subflow(
         Max temperature (K).
     phonopy_kwargs
         Additional kwargs to pass to the Phonopy class.
-    additional_fields
-        Additional fields to store in the database.
 
     Returns
     -------
     PhononSchema
         Dictionary of results from [quacc.schemas.phonons.summarize_phonopy][]
     """
 
-    phonon = get_phonopy(
-        atoms,
-        min_lengths=min_lengths,
-        supercell_matrix=supercell_matrix,
-        symprec=symprec,
-        displacement=displacement,
-        phonopy_kwargs=phonopy_kwargs,
-    )
-    supercells = [
-        phonopy_atoms_to_ase_atoms(s) for s in phonon.supercells_with_displacements
-    ]
-
     @subflow
     def _get_forces_subflow(supercells: list[Atoms]) -> list[dict]:
         return [
             force_job(supercell) for supercell in supercells if supercell is not None
         ]
 
     @job
-    def _thermo_job(atoms: Atoms, force_job_results: list[dict]) -> PhononSchema:
-        phonon = get_phonopy(
-            atoms,
-            min_lengths=min_lengths,
-            supercell_matrix=supercell_matrix,
-            symprec=symprec,
-            displacement=displacement,
-            phonopy_kwargs=phonopy_kwargs,
-        )
+    def _thermo_job(
+        atoms: Atoms,
+        phonopy: Phonopy,
+        force_job_results: list[dict],
+        t_step: float,
+        t_min: float,
+        t_max: float,
+    ) -> PhononSchema:
         parameters = force_job_results[-1].get("parameters")
         forces = [output["results"]["forces"] for output in force_job_results]
-        phonon_results = run_phonopy(
-            phonon, forces, t_step=t_step, t_min=t_min, t_max=t_max
+        phonopy_results = run_phonopy(
+            phonopy, forces, t_step=t_step, t_min=t_min, t_max=t_max
         )
 
         return summarize_phonopy(
-            phonon,
-            atoms,
-            phonon_results.directory,
-            parameters=parameters,
-            additional_fields=additional_fields,
+            phonopy, atoms, phonopy_results.directory, parameters=parameters
         )
 
     if relax_job is not None:
         atoms = relax_job(atoms)["atoms"]
 
+    phonopy = get_phonopy(
+        atoms,
+        min_lengths=min_lengths,
+        supercell_matrix=supercell_matrix,
+        symprec=symprec,
+        displacement=displacement,
+        phonopy_kwargs=phonopy_kwargs,
+    )
```

Comment:
@tomdemeyere: FYI --- It looks like `get_phonopy()` was originally being called _before_ the `relax_job`, which seems wrong. I have corrected that here.

Reply:
Oops, I didn't even notice it, nice catch